src/library/scala/collection/Iterable.scala

```diff
@@ -69,12 +69,8 @@ trait Iterable[+A] extends IterableOnce[A]
   protected[this] def stringPrefix: String = "Iterable"
 
   /** Converts this $coll to a string.
-    *
-    *  @return   a string representation of this collection. By default this
-    *            string consists of the `className` of this $coll, followed
-    *            by all elements separated by commas and enclosed in parentheses.
     */
-  override def toString = mkString(className + "(", ", ", ")")
+  override def toString: String  = className + "(<not computed>)"
```

Comment:
I don't think we can really say `<not computed>` as the default, it's more like in a state of `<unsure if computed or not>`. Maybe the Iterable should either not implement a toString, or should leave it as similar to `Iterator`'s implementation? (`iterator.toString` is  `<iterator>`).

Reply:
So `<iterable>`?

Reply:
That's correct 😄

Reply:
If you wanted to be fancy, you could report `"of $knownSize elements"` if non-negative.

I guess `ScalaRunTime.stringOf` only custom prints `StrictOptimizedIterable`, to pretty print `List[Array[String]]` for example. Otherwise `toString`.

Reply:
Changed to `<iterable>` in https://github.com/scala/scala/pull/8554/commits/e0c5211af040b41a829d156149288e7fab5ac76c